docs/root/modules/kubernetes/schema.md

```diff
@@ -108,3 +113,22 @@ Representation of a [Kubernetes Service.](https://kubernetes.io/docs/concepts/se
     ```
     (KubernetesService)-[SERVES_POD]->(KubernetesPod)
     ```
+
+### KubernetesSecret
+Representation of a [Kubernetes Secret.](https://kubernetes.io/docs/concepts/configuration/secret/)
+
+| Field | Description |
+|-------|-------------|
+| firstseen | Timestamp of when a sync job first discovered this node |
+| lastupdated | Timestamp of the last time the node was updated |
+| id | UID of the kubernetes service |
+| name | Name of the kubernetes service |
+| created\_at | Timestamp of the creation time of the kubernetes service |
+| deleted\_at | Timestamp of the deletion time of the kubernetes service |
+| type | Type of kubernetes service e.g. `Opaque` |
```

Comment:
`secret`, not `service`?

Reply:
Should be fixed (ecd2ff24845e08f98e3877c54f2e664afcb8780a)